workflows/templates/base/iufBase.template.argo.yaml

```
@@ -137,6 +137,10 @@ spec:
             touch {{inputs.parameters.script_output_file}}
           else
             bash -e /tmp/${ts}.sh
+            if [ $? -ne 0 ]; then
```

Check the exit status directly from the command:
```suggestion
            if ! bash -e /tmp/${ts}.sh; then
```